src/firebolt/client/resource_manager_hooks.py

```diff
@@ -7,32 +7,53 @@
 
 
 def log_request(request: Request) -> None:
-    """Hook to log http requests"""
+    """Log HTTP requests.
+
+    Hook for an HTTP client
+
+    Args:
+        request (Request): Request to log
+    """
     logger.debug(
-        f"Request event hook: {request.method} {request.url} - Waiting for response"
+        "Request event hook: %s %s - Waiting for response", request.method, request.url
```

Comment:
Is this the best practice, as opposed to f-strings, according to the checker?

Reply:
Yes, this way it performs a lazy evaluation